(Merge PR #43 first) Added page for the mentor and mentee profile lists

madeleinel opened the pull request:
I built on what you had put in app.py for the mentor list path, and used the same for the mentee list path > once the database is ready, the db paths for the mentee page will have to be updated (eg mentordict > menteedict).

## I have:

- Created the mentor and mentee list pages:
  - One HTML file for each page, one CSS file for both pages
- Restructured the structure of the HTML and CSS files you had created, to create a the basic layout and styling of the page which matches the rest of the site
- Added placeholder images for the profiles > will have to pull those from the Twitter API later (alternatively use placeholders like those you had put in the original mentorlist.html, if we don't want to use the photos eg for privacy reasons)

## Let me know what you think!

Eg if you want to change anything back to how you had styled it.

twinlensreflex committed:
added html and css files for mentorlist page

twinlensreflex committed:
add alternating gradients

twinlensreflex committed:
put random stuff to preview template

madeleinel committed:
Started creating the mentee and mentor profile pages: So far have just set up the paths in app.py and created basic HTML files

madeleinel committed:
Added the basic structure of the mentor profile page

madeleinel committed:
Created the basic layout and styling of the mentor profile page

madeleinel committed:
applied the same styling to the mentee profile page (still need to update mentee paths within app.py; to be done when the database is ready)

twinlensreflex commented:
hey,
re:

> Added placeholder images for the profiles > will have to pull those from the Twitter API later (alternatively use placeholders like those you had put in the original mentorlist.html, if we don't want to use the photos eg for privacy reasons)

did you actually check how to do that? because I think it's a whole lot more code for something fairly minimal we could just use a normal placeholder for, we'd have to request the API every time someone goes to the page

madeleinel commented:
No, I just remembered we'd talked about doing that when we were planning the project. But yeah I think at this point using a placeholder is probably the best course of action.

I'll update that and then resolve the issues so this can be merged.

madeleinel committed:
added links for the mentee and mentor profile lists to the homepage

madeleinel committed:
Updated the example tweets in app.py

madeleinel committed:
Cleaned up the CSS files, removed commented out code. Added example mentor tweets to the mentor signup page

madeleinel committed:
added CSS prefixes where necessary

madeleinel committed:
delete archived files

madeleinel committed:
Merge branch 'master' into feature/profile-pages

madeleinel commented:
## I have now:

- Resolved the PR conflicts
- Added links for the mentee and mentor profile lists to the homepage
- Updated the example tweets in app.py
- Cleaned up the CSS files:
  - Removed commented out code
  - added CSS prefixes where necessary
- Added example mentor tweets to the mentor signup page
- Added new profile photo placeholders (silhouetted head)
